src/test/java/org/ays/tests/institution/assignmentmanagementservice/PostAssignmentTest.java

```diff
@@ -35,7 +35,7 @@ public void createAnAssignmentPositive() {
 
     }
 
-    @Test(dataProvider = "invalidDescriptionData")
+    @Test(dataProvider = "invalidDescriptionData",dataProviderClass = DataProvider.class)
```

Comment:
Kod formatında problem var. Bu tür durumlar yaşamamak için aşağıdaki dokümanı inceleyebilirsiniz;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html